lib/puppet/pops/serialization/extension.rb

```diff
@@ -26,8 +27,8 @@ module Extension
   TIMESPAN = 0x34
   VERSION = 0x35
   VERSION_RANGE = 0x36
-  SENSITIVE = 0x37
-  BINARY = 0x38
+  BINARY = 0x37
+  BASE64 = 0x38
```

Comment:
Why do we have a BASE64 serialization type? Isn't that a choice per serializer how to treat a Binary ?